server/store/adapters/rdbms/upgrade_fixes.go

```diff
@@ -1021,6 +1022,65 @@ func fix_2024_09_03_addFederationNodeSyncNodeIDIndex(ctx context.Context, s *Sto
 	return s.DataDefiner.IndexCreate(ctx, "federation_nodes_sync", &nodeIDIndx)
 }
 
+func fix_2024_12_00_addRevisionDraftColumns(ctx context.Context, s *Store) (err error) {
+	revisionColumns := []*dal.Attribute{
+		{Ident: "resource_type", Type: &dal.TypeText{Nullable: true}},
+		{Ident: "status", Type: &dal.TypeText{Nullable: true}},
+		{Ident: "deleted_at", Type: &dal.TypeTimestamp{Nullable: true}},
+		{Ident: "deleted_by", Type: &dal.TypeID{Nullable: true}},
+	}
+
+	tables := []string{"compose_record_revisions", "system_revisions"}
```

Comment:
You can grab this info from the generated resource models based on .cue files; I can't recall exactly where they are, though....